modules/islandora_iiif/src/Plugin/views/style/IIIFManifest.php

```diff
@@ -232,7 +246,7 @@ public function render() {
       // For each row in the View result.
       foreach ($this->view->result as $row) {
         // Add the IIIF URL to the image to print out as JSON.
-        $canvases = $this->getTileSourceFromRow($row, $iiif_address, $iiif_base_id);
+        $canvases = $this->getTileSourceFromRow($row, $iiif_address, $iiif_base_id, $structured_text_term);
```

Comment:
This is no longer accepted by the called method.

```suggestion
        $canvases = $this->getTileSourceFromRow($row, $iiif_address, $iiif_base_id);
```